### ISSUE TITLE
Warp Seems Bugged

### ISSUE THREAD

**Issue by DarkLordIce** [nutsandboltspack/nutsandbolts#33]
I have not received a single warp effect since the update, and I have pretty much maxxed out warp, used to get an effect at least once or twice every 10 min.

Unsure if its a bug, or it got reset w/o the sanity checker realizing (it sill shows me full).

So, yea /reported!

**Comment by 666lumberjack** [comment]
The new Thaumcraft decreases the frequency of warp effects based on the time since you last gained warp. Is it possible you haven't gained any warp recently so you're not getting anything?

**Comment by DarkLordIce** [comment]
That is possible, though I did do a few infusions that were marked as bad.  I'll try and purposely gain some more warp and test it.
Just seemed strange going from getting some sort of warp effect every few minutes to none for hours.

Though, late last night well after I submitted this, I did get the heart-pounding sound and gained several research points, but no negative effects.  Its the only warp effect I have experienced since the update rolled out.
